build-system/tasks/performance/index.js

```diff
@@ -32,13 +33,17 @@ async function performance() {
   await compileScripts(urls);
   await rewriteScriptTags(urls);
   await getMetrics(urls, {headless, runs});
+  runTests();
   printReport(urls);
 }
 
 performance.description = '  Runs web performance test on current branch';
```

Comment:
Nit: To make `gulp help` more readable, remove the two spaces before `performance.description` and add two spaces before each string in `performance.flags`. (It's our low-tech way of indenting flag descriptions.)